parser/raylib_parser.c

```diff
@@ -743,13 +764,19 @@ static bool IsTextEqual(const char *text1, const char *text2, unsigned int count
     return result;
 }
 
-// Search and replace a character within a string.
-static char* CharReplace(char* text, char search, char replace)
+// Escape backslashes in a string, writing the escaped string into a static buffer
+static char* EscapeBackslashes(char *text)
 {
-    for (int i = 0; text[i] != '\0'; i++)
-        if (text[i] == search)
-            text[i] = replace;
-    return text;
+    static char buf[256];
+    char *a = text;
+    char *b = buf;
+    do
```

Comment:
I usually try to avoid the `do-while` formats. I can review that function later on.